tests/pull-update/test.bats

```diff
@@ -6,6 +6,21 @@ setup_file() {
 
 	# Copy all the files there
 	cp -r . "${TEMP_DIR}"
+
+	# On Windows (MSYS2/Git Bash), convert to mixed-style path for conftest compatibility
+	if command -v cygpath >/dev/null 2>&1; then
+		TEMP_DIR=$(cygpath -m "${TEMP_DIR}")
+	fi
+
+	# Create a git repository for the remote-policy to enable git:// URL downloads
+	# This avoids the file:// symlink issues on Windows
```

Comment:
Please revert this. The test should be able to run without forcing the Git protocol.